code/modules/assembly/bomb.dm

```diff
@@ -166,7 +168,7 @@
 	else if(bomb_mixture.temperature > (T0C + 250))
 		strength = (fuel_moles/20)
 
-		if(strength >=1)
+		if (strength >=1)
```

Comment:
why

Reply:
Oh jesus christ, thats a random bad that I did.